index.html

```diff
@@ -0,0 +1,331 @@
+  <dt markdown>`version: "v1.0"`</dt>
+  <dd markdown>The tag for the [version](#versioning) of the spec used to author the document. Processors **must** select an implementation of the spec which can [satisfy](#satisfaction) the specified version.</dd>
+</dl>
+
+The version tag **must** satisfy the [BNF grammar for `‹version tag›`](#backus-naur-form-for-versions-and-version-tags).
+
+### Ignore meaningless URL components
+
+When extracting the URL's `prefix` and `version`, processors **must** ignore any url components which are not assigned a meaning. This spec assigns meaning to the final two components of the [path](https://tools.ietf.org/html/rfc3986#section-3.3). Other URL components—particularly query strings and fragments, if present—**must** be ignored for the purposes of extracting the `prefix` and `version`.
+
+<figure>
+  <figcaption>
+    <a name=spec-url-ignore-meaningless href=#spec-url-ignore-meaningless class=a-header>Ignoring meaningless parts of a URL</a>
+  </figcaption>  
+  <code class=anatomy>
+    <span class=pink style='--depth: 2'>https://example.com/<span>exampleSpec<aside>prefix</aside></span><aside>
+      identity
+    </aside></span>/<span style='--depth: 2' class=green>v1.0<aside>version</aside></span><span class=grey>?key=val&k2=v2#fragment<aside>ignored</aside></span>
+  </code>
+</figure>
+
+### Why is versioning in the URL, not a directive argument?
+
+The version is in the URL because when a human reader visits the URL, we would like them to be taken to the documentation for the *version of the specification in use by this document*. Many text editors will turn URLs into hyperlinks, and it's highly desirable that clicking the link takes the user to the correct version of the docs. Putting the version information in a separate argument to the `@using` directive would require this.
+
+## Prefixing
+Specifications meant to be referenced via `@using` **should** prefix any schema elements they introduce. The prefix **must** be a string of characters valid within GraphQL typenames, with the exception of `"_"`.
+
+Specs **should** define any new schema elements with names that start with `prefix_`.
+
+```graphql
+#::[Using the `@using` directive with a spec's default prefix]
+schema @using(spec: "https://spec.example.com/example/v1") {
+  query: Query
+}
+
+type User {
+  name: String @example_someDirective
+               # prefixed directive from the example spec
+}
+```
+
+The language of the spec **should** specify the default prefix and **should** identify locations where the prefix is used. The prefix **should not** be elided within documentation; definitions of schema elements provided within the spec **must** include the default prefix.
+
+### Elements which should be prefixed
```

Comment:
I have a general concern here that there's not enough guidance on how schema processors/consumers should utilize these rules/constraints (presumably this would be in section 9 "Processing Schemas"). More specifically, when a processor/consumer receives a schema that uses the `@using` spec, it's a bit unclear how it should proceed, such as the validations it should perform, or how it should determine which spec(s) govern a directive/type/field/argument/enum value. 

For example, when encountering a type/directive with a prefix, should the consumer validate that the prefix be declared in some `@using` directive on the schema? If there is a corresponding `@using` directive, should the consumer validate that the schema element is one that has been introduced by the corresponding spec? (E.g. imagine encountering  `cs_foobar`.) How would those answers differ for fields/arguments/enum values, where we introduce unprefixed elements that can be associated with the spec of the parent?

I'm guessing we're going to punt these questions until the implementation and let that inform the spec, which is fine, but I also have worries that the implementation might be optimized for the narrower use case we're designing them for (CSDL), and may not reveal or resolve issues that arise with multiple specs unless we take the effort to think about and build tests for them. (The answer to this might be that we're fine with the implementation being narrowly-tailored, and we'll just avoid bumping the `@using` spec to `v1.0` until we've encountered enough use cases to arrive at a more complete implementation.)

Reply:
I've added one validation, re: prefix uniqueness. There's another possible strict validation we could specify, in which everything that `looks_prefixed` is bound to an actual spec with `@using`, but I'd love to figure out a way to get that to play nicer with people who have regrettably used underscores in their graph names to begin with.

But more generally, `@using` and any validations we specify here aren't meant to catch every wrong thing you can do in a document. For processors, its primary use is to tell them what spec implementations to activate. Those spec implementations will then presumably perform more stringent validations of their own.